skull-king-app/public/bonus-test-updated.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
```

Comment:
I don't like what this commit did to the UI. please revert 9f7c6542faafb0dfc1e953e4eef553eeeb8fb494